solr_conf_4_testing/schema.xml

```diff
@@ -144,12 +145,13 @@
     <field name="pub_country" type="text" indexed="true" stored="true" omitNorms="true"/>
     <!-- TODO: should pub_date_search be a date or a text field? -->
     <field name="pub_date_search" type="text" indexed="true" stored="true" omitNorms="true"/>
-    <!-- pub_date_sort now deprecated; replaced by pub_year_isi -->
```

Comment:
spotlight should switch to pub_year_isi for sorting.  It is an improved implementation in stanford-mods.